24H Study Room/Protocols/FirestoreServicing.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  FirestoreServicing.swift
+//  24H Study Room
+//
+//  Created by Jia-Jiuan Tsai on 2022-08-30.
+//
+
+import Foundation
+
+protocol FirestoreServicing {
+    func fetchOnlineClassmates()
```

Comment:
This method has no completion closure or return value, and this protocol doesn't expose any state separately either. Therefore, there is no way to obtain data through this protocol.